src/PowerShellEditorServices/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -27,63 +27,103 @@ public class PesterDocumentSymbolProvider : FeatureProviderBase, IDocumentSymbol
                 return Enumerable.Empty<SymbolReference>();
             }
 
-            var commandAsts = scriptFile.ScriptAst.FindAll(ast =>
-            {
-            CommandAst commandAst = ast as CommandAst;
+            // Find plausible Pester commands
+            IEnumerable<Ast> commandAsts = scriptFile.ScriptAst.FindAll(IsNamedCommandWithArguments, true);
+
+            return commandAsts.OfType<CommandAst>()
```

Comment:
can `commandAsts` ever be null on accident? It looks like there was that `commandant != null` check previously

Reply:
`FindAll` promises to return a possibly empty collection, but not null